windows/vtpm_cold_add_remove/bitlocker_enable_test.yml

```diff
@@ -50,6 +50,12 @@
       ansible.builtin.set_fact:
         win_enable_bitlocker_cmd: "Enable-BitLocker -MountPoint 'C:' -UsedSpaceOnly -TpmProtector -SkipHardwareTest"
 
+    - name: "Workaround for known issue"
+      include_tasks: handle_vm_known_issue.yml
+      when:
+        - guest_os_ansible_distribution is search('10|11|2019|2022|2025')
```

Comment:
Is there only version number in guest_os_ansible_distribution? Or could these versions be in any other numbers?
I suggest extracting the version number and check it in a list as below:
```
guest_os_ansible_distribution | regex_search('\\d+') in ['10', '11', '2019', '2022', '2025']
```